src/specs/ViewModel.nitro.ts

```diff
@@ -124,3 +127,28 @@ export interface ViewModelImageProperty
   /** Add a listener to the view model image property */
   addListener(onChanged: () => void): void;
 }
+
+/**
+ * A list property that contains a dynamic collection of {@link ViewModelInstance} objects.
+ * @see {@link https://rive.app/docs/runtimes/data-binding#lists Rive Data Binding Lists}
+ */
+export interface ViewModelListProperty
```

Comment:
Two things to consider:
- We don't HAVE to name these similar to android/ios. We can use the naming convention that matches what is expected when working with lists in RN.
- We could add additional methods/overloads if those are commonly used in RN. For example in Dart I added `firstOrNull` instead of throwing, and overloads for `operator []` and `operator[]=`. Maybe something like this possible here to

```
  @override
  ViewModelInstance operator [](int index) => instanceAt(index);

  @override
  void operator []=(
    int index,
    covariant FFIRiveViewModelInstanceRuntime value,
  ) =>
      insert(index, value);
```